pyresample/geometry.py

```diff
@@ -547,6 +547,39 @@ def __init__(self, lons, lats, nprocs=1):
         elif lons.ndim > 2:
             raise ValueError('Only 1 and 2 dimensional swaths are allowed')
 
+    def copy(self):
+        return SwathDefinition(self.lons, self.lats)
+
+    def aggregate(self, **dims):
```

Comment:
Docstring? And is this another function that won't work in python 2?

Reply:
The aggregation in this case doesn't need xarray's coarsen, so it should work with python 2 too.

Reply:
Fixed